src/main/java/org/project_kessel/relations/client/HealthClient.java

```diff
@@ -8,14 +8,14 @@
 import org.project_kessel.api.relations.v1.GetReadyzResponse;
 import org.project_kessel.api.relations.v1.GetReadyzRequest;
 import org.project_kessel.api.relations.v1.KesselHealthServiceGrpc;
+import org.project_kessel.api.relations.v1beta1.KesselCheckServiceGrpc;
+import org.project_kessel.clients.KesselClient;
 
-public class HealthClient {
+public class HealthClient extends KesselClient<KesselHealthServiceGrpc.KesselHealthServiceStub, KesselHealthServiceGrpc.KesselHealthServiceBlockingStub> {
```

Comment:
We probably don't need to keep the health service stuff in the client anymore. I remember we were talking about removing it as its not necessary. Otherwise it looks good!

Reply:
@lennysgarage why?

Reply:
We can talk about it again, I debated taking it out in this PR because @alechenninger reckoned it shouldn't have been in there in the first place, but I just wanted to keep this PR about refactoring.